deepchem/feat/materials_featurizers.py

```diff
@@ -0,0 +1,178 @@
+"""
+Featurizers for inorganic crystals.
+"""
+
+import numpy as np
+
+from deepchem.feat import Featurizer
+from deepchem.utils import pad_array
+
+from matminer.featurizers.composition import ElementProperty
+
+from pymatgen import Composition, Structure
+from pymatgen.analysis.graphs import StructureGraph
+from pymatgen.analysis.local_env import MinimumDistanceNN
+
+
+class ChemicalFingerprint(Featurizer):
+  """
+  Chemical fingerprint of elemental properties from composition.
+
```

Comment:
Can you expand out the docstring here to add more details about the method? This should include MLA style citations to any papers that provide inspirations, a mention of the pymatgen depdency for this featurizer and a general discussion of when this would be useful.